### PR TITLE
add missing option input config

### LINKED ISSUE #20: Require auto merge enabled is not working as documented
I'm getting the following error: `Invalid action input 'require_auto_merge_enabled'`
I can see from the PR enabling `require_auto_merge_enable` that it's not including the right input for auto merge enable.
`require_auto_merge_anable` should be defined as an input in this file:
image

If allowed, I could contribute to fix the issue.

### PR DESCRIPTION
**Pull request by zhiyelee**
Add the missing config for recently added option `require_auto_merge_enabled`
close #20

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -19,12 +19,16 @@ inputs:
     required: false
     description: 'If the action should skip PRs that have failed checks, defaults to `true`'
     default: 'true'
+  require_auto_merge_enabled:
+    required: false
+    description: 'If the action should skip PRs that have failed checks, defaults to `true`'
```
Comment (nickqi-tubi) [inline action.yml:24]: Was this description copied and pasted from `require_passed_checks`? Please update it accordingly.

**Review by nickqi-tubi — DISMISSED** [review]
Please update the description of `require_auto_merge_enabled`. Other changes look good to me.

File: action.yml (hunk 1 above)
Comment (zhiyelee) [reply action.yml:24]: @nickqi-tubi thanks for the catch. Copilot did that, I thought it was correct

Comment (zhiyelee) [reply action.yml]: Updated, thanks

**Review by nickqi-tubi — APPROVED** [review]

**Pull request merged**